### DISCUSSION TITLE
Unity Windows Desktop Application

### DISCUSSION THREAD

**Discussion by Valkynen** [phantom/docs#267] [Help & Support] [answered]
### Summary

I am developing a game in Unity for Windows for which I'd like to add phantom wallet support.  Specifically for Solana.  I am currently looking at Magicblock's Solana SDK for Unity. (https://github.com/magicblock-labs/Solana.Unity-SDK).  Unfortunately they do not support Phantom in Windows standalone builds yet'  Only in webgl and mobile.  My understanding is that is not on their short term roadmap.  I was wondering if anybody here could provide some guidance,

### Steps to Reproduce

This is not a bug with Phantom specifically

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hey @Valkynen,

Thanks for the question. I was not able to find any current libraries that support connect to phantom from a windows build. The extension provider is not available outside of the browser context.

We have recently announced that we are building embedded wallet it is possible that will be an option for your users to connect to their phantom wallet once available. https://x.com/phantom/status/1792903231774834689

**Reply by Valkynen** [reply]
Interesting, is there a timeline for release?

**Reply by adamdelphantom** [reply]
We have not yet released a time line. Please stay tuned to the Phantom twitter account for coming details.
